GoogleForCreators/web-stories-wp
# Moved keyboard shortcuts to left side

## Context

This moves the keyboard shortcuts icon and its accompanying popup to the left side of the workspace.

## Relevant Technical Choices

* Converts the existing carousel menu (on the right side) to a primary menu
* Adds a secondary menu component to the left side and moves the help center and keyboard shortcuts button inside it
* Fixes some RTL weirdness around padding

## Todo

- [x] Fix bug with help center open at the same time as shortcuts dialog.
- [ ] Fix broken test(s)

## User-facing changes

| What | Before | After |
|-|-|-|
| LTR, menu closed | ![Screen Shot 2021-03-29 at 17 16 38](https://user-images.githubusercontent.com/637548/112901805-39e84080-90b3-11eb-920b-a293ce334279.png) | ![Screen Shot 2021-03-29 at 17 17 27](https://user-images.githubusercontent.com/637548/112901815-3d7bc780-90b3-11eb-967b-397ea26be095.png) |
| LTR, menu open | ![Screen Shot 2021-03-29 at 17 16 43](https://user-images.githubusercontent.com/637548/112901780-31900580-90b3-11eb-9963-85a0a1c6e698.png) | ![Screen Shot 2021-03-29 at 17 17 33](https://user-images.githubusercontent.com/637548/112901793-348af600-90b3-11eb-83a9-b55fd83864c8.png) |
| RTL, menu closed | ![Screen Shot 2021-03-29 at 17 16 03](https://user-images.githubusercontent.com/637548/112901741-250bad00-90b3-11eb-9a93-37b382b796f3.png) | ![Screen Shot 2021-03-29 at 17 15 21](https://user-images.githubusercontent.com/637548/112901746-26d57080-90b3-11eb-8c81-5bf2f3cfd8d1.png) |
| RTL, menu open | ![Screen Shot 2021-03-29 at 17 16 10](https://user-images.githubusercontent.com/637548/112901618-00173a00-90b3-11eb-98a4-62f90cc58bbf.png) | ![Screen Shot 2021-03-29 at 17 15 15](https://user-images.githubusercontent.com/637548/112901724-1cb37200-90b3-11eb-9d1b-15f99d229061.png) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Open the keyboard shortcuts dialog and observe that it looks good
2. Now open the help center and observe that the keyboard shortcuts dialog is closed
3. Now open the keyboard shortcuts dialog and observe that the help center is closed

### UAT




## Checklist


---

Fixes #6381
Fixes #6413
